package.json

```diff
@@ -113,6 +114,7 @@
                     "args": "none"
                 }
             ],
+            "object-curly-spacing": "off",
```

Comment:
@YuHsuan-Hwang this solves the problem of eslint reformatting things when it rearranges imports